README.md

```diff
@@ -1,56 +1,14 @@
-[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
 
-# AWS Control Tower Customization for Integration with New Relic
+## New Relic AWS Control Tower Integration
 
-> This solution helps you automate setup of [New Relic's AWS integration](https://newrelic.com/aws) in your [AWS Control Tower](https://aws.amazon.com/controltower/) managed multi-account environment (landing zone). Once the solution is deployed to your [AWS Control Tower management account](https://docs.aws.amazon.com/controltower/latest/userguide/how-control-tower-works.html#what-is-master), any new accounts you enroll in your landing zone are automatically integrated with New Relic.
+For architectural details, step-by-step instructions, and customization options, see the [deployment guide](https://fwd.aws/3JXVA?).
 
-## Installation
+To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
 
-> The repo includes a couple of [AWS CloudFormation](https://aws.amazon.com/cloudformation/) templates that you are free to download or directly reference their public GitHub URL when trying to create CloudFormation stacks using them, as described in [Usage](#Usage)
+To submit code for this Quick Start, see the [AWS Quick Start Contributor's Guide](https://aws-quickstart.github.io/).
 
-## Prerequisites
-> Fully deployed AWS Control Tower is required for this solution. You will need administrator privileges in the AWS Control Tower management account to deploy the solution. For information about setting up an AWS Control Tower landing zone, see [Getting Started with AWS Control Tower](https://docs.aws.amazon.com/controltower/latest/userguide/getting-started-with-control-tower.html) in the AWS Control Tower User Guide.
+### Customization Framework for Control Tower (CfCT) Support
```

Comment:
I would suggest to add this section to the guide and not on the readme. Readme's have standard format which we don't want to deviate from.

Reply:
Agree, there will be separate PR to address the doc guide.

I was copying readme pattern from other quickstart with CfCT support: https://github.com/aws-quickstart/quickstart-mongodb#deploy-with-control-tower 

if its not needed, happy to take it off.

Reply:
readme updated